frontend/web/components/Tooltip.tsx

```diff
@@ -44,7 +45,7 @@ const Tooltip: FC<TooltipProps> = ({
           ) : (
             <div
               style={{ wordBreak: 'break-word' }}
-              dangerouslySetInnerHTML={{ __html: children }}
+              dangerouslySetInnerHTML={{ __html: sanitize(children) }}
```

Comment:
I don't really understand why we're even using `dangerouslySetInnerHTML` here at all? Shouldn't we be able to just render out the text as plain text?